commerce_coordinator/settings/local.py

```diff
@@ -78,11 +78,11 @@
 )
 BACKEND_SERVICE_EDX_OAUTH2_KEY = os.environ.get(
     'BACKEND_SERVICE_EDX_OAUTH2_KEY',
-    'commerce-coordinator-backend-service-key'
+    'commerce_coordinator-backend-service-key'
 )
 BACKEND_SERVICE_EDX_OAUTH2_SECRET = os.environ.get(
     'BACKEND_SERVICE_EDX_OAUTH2_SECRET',
-    'commerce-coordinator-backend-service-secret'
+    'commerce_coordinator-backend-service-secret'
```

Comment:
We run the commerce coordinator's service locally using the `local.py` settings file as mentioned in the [REAMDE](https://github.com/edx/commerce-coordinator?tab=readme-ov-file#local). Would this change affect all the existing developers who have already run the provisioning script that creates the service user locally using this OAUTH key and secret? (Ref: https://github.com/edx/commerce-coordinator/blob/f0570c3a6d4b0fe1671c207872e2096d44f121a3/provision-commerce-coordinator.sh#L54)